[release-2.4.0 < T0807] Adding changelog for Memory allocator issue. (#427)
memgraph/docs: docs/changelog.md

- Fix header on `SHOW REPLICATION ROLE` query and wrong timout info on `SHOW
  REPLICAS` query. [#376](https://github.com/memgraph/memgraph/pull/376)

> - Adapted compilation flag so that the memory allocator to use JEMALLOC to count allocated memory. [#401](https://github.com/memgraph/memgraph/pull/401)

Reviewer: ```suggestion
- Adapted compilation flag so that the memory allocator uses JEMALLOC while counting allocated memory. [#401](https://github.com/memgraph/memgraph/pull/401)
```
